src/api/.env.sample

```diff
@@ -34,3 +34,5 @@ GOV_NOTIFY_EMAIL_RECIPIENT_1=
 GOV_NOTIFY_EMAIL_RECIPIENT_2=
 FEEDBACK_EMAIL_RECIPIENT=
 UNDERWRITING_TEAM_EMAIL=
+
+ENCRYPTION_KEY=
```

Comment:
Should we be more descriptive here?

Thinking if we encrypt other things, we might want a different key. Maybe could be called `LOSS_PAYEE_ENCRYPTION_KEY`

@abhi-markan ?

Reply:
changed